packages/client/src/components/pages/home-page/home-page.tsx

```diff
@@ -140,6 +141,7 @@ export const HomePage = () => {
       }
 
       try {
+        await dispatch(OrphanedRecordActions.fetchCount({ orgId: orgId! }));
         await dispatch(OrphanedRecordActions.fetchPage({
```

Comment:
Can this `fetchPage` call be removed? It looks like that may have been there as a means of getting `totalRowsCount` for the orphaned records count, which shouldn't be necessary now.

Reply:
done